tests/src/FunctionalJavascript/WebformCivicrmTestBase.php

```diff
@@ -300,7 +300,7 @@ protected function editContactElement($params) {
     $contactElementEdit = $this->assertSession()->elementExists('css', "[data-drupal-selector=\"{$params['selector']}\"] a.webform-ajax-link");
     $contactElementEdit->click();
     $this->htmlOutput();
-    $this->assertSession()->assertWaitOnAjaxRequest();
+    $this->assertSession()->assertWaitOnAjaxRequest($timeout = 20000);
     $this->assertSession()->elementExists('css', '[data-drupal-selector="edit-form"]')->click();
```

Comment:
We can remove this - I was just trying to see if we needed to increase the timeout on this assertWait in 9.2.0

Reply:
Ok removing this now